aws_lambda_powertools/event_handler/appsync.py

```diff
@@ -126,7 +126,7 @@ def handler(event, context: LambdaContext):
 
         class MyCustomModel(AppSyncResolverEvent):
             @property
-            def country_viewer(self) -> str:
+            def country_viewer(self):
```

Comment:
isn't that str?

Reply:
No. That's actually returning `Optional[str]`. I thought it would be simpler to just omit the type. Let me know if I should add ` -> Optional[str]` and the respective Optional import.